scripts/get-channels-utils.ts

```diff
@@ -0,0 +1,14 @@
+import {Versions} from '../configs/schemas/versions';
+
+export function getChannels(
+  ampVersion: string,
+  currentVersions: Versions
+): string[] {
+  const channels = [];
+  for (const [channel, version] of Object.entries(currentVersions)) {
+    if (version && version.slice(-13) == ampVersion) {
+      channels.push(channel);
+    }
+  }
+  return channels;
```

Comment:
How about:
```suggestion
  return Object.entries(currentVersions)
      .filter(([channel, version]) => version?.slice(-13) == ampVersion)
      .map(([channel,]) => channel);
```

Reply:
sure thanks! (didn't commit directly bc prettier and lint would complain)

Reply:
Fair :D